_rules/SC2-4-4+2-4-9+4-1-2-link-has-name.md

```diff
@@ -197,6 +213,30 @@ Not included in the accessibility tree.
 <a href="http://www.w3.org/WAI" aria-hidden="true"><img src="#" /></a>
 ```
 
+#### Failed example 9
+
+Link is completely empty, but still shows up in focus order, so it should have an accessible name.
+
+```html
+<a href="http://www.w3.org/WAI"></a>
+```
+
+#### Failed example 10
+
+`area` element with `href` attribute does not have accessible name.
+
+```html
+<area shape="rect" coords="0,0,82,126" href="http://www.w3.org/WAI">
```

Comment:
As above, this element can't be reached by AT if it doesn't map to an image.

Reply:
Related to https://github.com/act-rules/act-rules.github.io/pull/430/files#r280081504:

I can easily navigate to this using arrow navigation in VoiceOver.
It has the role of link, but no accessible name, so I think it is a perfectly valid test case.